### ISSUE TITLE
ReferenceError: global is not defined

### ISSUE THREAD

**Issue by sholladay** [jessepollak/payment#117]
`Payment` assigns to `global.Payment`, but `global` is not available in the browser and causes an exception.

https://github.com/jessepollak/payment/blob/f167412da35964ad3122231f7289b829b0b3491d/src/index.coffee#L551

I propose changing that line to use [`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) instead of `global`. The difference is that `globalThis` works in both browsers and Node.js, whereas `global` only works in Node.js.

**Comment by melloware** [comment]
As soon as the build in your PR is done I will merge this.

**Comment by sholladay** [comment]
Thanks for the fast response. Unfortunately, Travis CI appears to be stuck.

**Comment by melloware** [comment]
Yeah and the Merge button is disabled until the Travis Build finishes.  maybe @jessepollak can turn off the requirement for the Travis build to complete before we can merge?

**Comment by melloware** [comment]
Looks like a unit test failed

**Comment by sholladay** [comment]
I looked into it. The reason is because PhantomJS, which is used in the tests, is ancient and [unmaintained](https://github.com/ariya/phantomjs/issues/15344). It doesn't know about `globalThis`. But `globalThis` does have great support in real environments, going all the way back to Node 12 and Chrome 71, which came out more than two years ago.

The change should be extremely safe and backwards compatible, that's why I chose that approach.

I could attempt to modernize the testing infrastructure in this project, although I'm not familiar with CoffeeScript.

**Comment by melloware** [comment]
Well the "Merge" button is locked out on me because the tests are passing. It will only let me Merge once the build succeeds.   But I don't want to comment out all tests. Hmmm.
